app/validators/questionnaire_validator.py

```diff
@@ -132,10 +132,10 @@ def validate_html(self):
             self.check_invalid_html_tags(html_strings)
 
     def check_invalid_html_tags(self, html_strings):
-        strong = re.compile(r"<strong>(?:(?!<\/strong>).)*</strong>")
-        anchor = re.compile("<a .*>.*</a>")
+        strong = re.compile(r"<strong>(?:(?!</strong>).)*</strong>")
+        anchor = re.compile("<a .*>(?:(?!</a).)*</a>")
```

Comment:
Raw string missing from the anchor expression.

Reply:
Updated 👍